packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # @redocly/openapi-core
 
+## 1.0.2
```

Comment:
Is this expected to be empty?

Reply:
This is because the packages versions are linked and there was no changes in the openapi-core itself. We can manually put something like `No code changes` but I don't see too much benefits of this as this changelog is not going to be published in our docs.